composer/src/lib.rs

```diff
@@ -455,13 +543,13 @@ mod tests {
     #[rstest]
     fn test_composer_json(#[files("tests/fixtures/*.json")] path: PathBuf) {
         let composer_json = fs::read(&path).unwrap();
-        serde_json::from_slice::<ComposerRootPackage>(&composer_json).unwrap();
+        dbg!(serde_json::from_slice::<ComposerRootPackage>(&composer_json).unwrap());
     }
 
     #[rstest]
     fn test_composer_lock(#[files("tests/fixtures/*.lock")] path: PathBuf) {
         let composer_lock = fs::read(&path).unwrap();
-        serde_json::from_slice::<ComposerLock>(&composer_lock).unwrap();
+        dbg!(serde_json::from_slice::<ComposerLock>(&composer_lock).unwrap());
```

Comment:
I'm not sure the purpose of these print statements. Can we remove them or annotate why they're here?

```suggestion
        serde_json::from_slice::<ComposerLock>(&composer_lock).unwrap();
```